src/kerberos_sspi.c

```diff
@@ -338,8 +355,8 @@ auth_sspi_client_step(sspi_client_state* state, SEC_CHAR* challenge) {
         status = AUTH_GSS_CONTINUE;
     }
 done:
-    if (inBufs[0].pvBuffer) {
-        free(inBufs[0].pvBuffer);
+    if (inBufs[1].pvBuffer) {
```

Comment:
If the caller doesn't provide channel bindings, the data to free will be at inBufs[0].